ModelicaReference/package.mo

```diff
@@ -3048,7 +3048,9 @@ Connections.branch(A.R,B.R);
 
 <h4>Description</h4>
 <p>
-Defines a non-breakable branch from the overdetermined type or record instance <code>R</code> in connector instance <code>A</code> to the corresponding overdetermined type or record instance <code>R</code> in connector instance <code>B</code> for a virtual connection graph.
+Defines a branch from the overdetermined type or record instance <code>R</code> in connector instance <code>A</code> to the corresponding overdetermined type or record instance <code>R</code> in connector instance <code>B</code> for a virtual connection graph.
+These branches are required to be part of the spanning-tree for the virtual connection graph (they do not directly generate equations, but should be combined with equations coupling A.R to B.R), 
+whereas connect-statements are optional for the spanning-tree (and generate different equations depending on whether they are part of the spanning-tree or not).
```

Comment:
`A.R to B.R` should be changed to `<code>A.R</code> to <code>B.R</code>"

Reply:
Updated as indicated.